chroma/chromakeying.php

```diff
@@ -32,48 +33,48 @@
 $chromaKeying = true;
 $GALLERY_FOOTER = true;
 
-include($fileRoot . 'template/components/main.head.php');
+include PathUtility::getAbsolutePath('template/components/main.head.php');
 ?>
 
 <body data-main-image="<?=$mainimage?>">
-	<div class="chromawrapper rotarygroup">
-	<?php if ($keying_possible): ?>
-		<div class="canvasWrapper <?php echo $uiShape; ?> noborder initial">
-			<canvas class="<?php echo $uiShape; ?>" id="mainCanvas"></canvas>
-		</div>
+    <div class="chromawrapper rotarygroup">
+    <?php if ($keying_possible): ?>
+        <div class="canvasWrapper <?php echo $uiShape; ?> noborder initial">
+            <canvas class="<?php echo $uiShape; ?>" id="mainCanvas"></canvas>
+        </div>
 
-		<div style="padding-top:10px;text-align:center;">
-		<?php
-            $dir = $fileRoot . $config['keying']['background_path'] . DIRECTORY_SEPARATOR;
-	    $cdir = scandir($dir);
-	    foreach ($cdir as $key => $value) {
-	        if (!in_array($value, ['.', '..']) && !is_dir($dir . $value)) {
-	            echo '<img src="' . $dir . $value . '" class="backgroundPreview ' . $uiShape . ' rotaryfocus" onclick="setBackgroundImage(this.src)">';
-	        }
-	    }
-	    ?>
-		</div>
+        <div style="padding-top:10px;text-align:center;">
+        <?php
+            $dir = PathUtility::getAbsolutePath($config['keying']['background_path']);
```

Comment:
Path must be part of Photobooth, used on html and therefore not visible.

![Uploading PXL_20230922_051712961.jpg…]()